kubetest/dind/dind.go

```diff
@@ -98,6 +98,7 @@ func (d *Deployer) BuildTester(o *e2e.BuildTesterOptions) (e2e.Tester, error) {
 	// dind tester sets parallelism a little differently
 	t.GinkgoParallel = 1
 	t.NumNodes = o.Parallelism
+	t.Provider = "skeleton"
```

Comment:
@BenTheElder should we set this statically? I can't imagine someone using a value other than Skeleton

Reply:
I think this makes sense. I'll try to follow up for something more thorough when kind integrates with kubetest